app/celer-sim/Transporter.cc

```diff
@@ -127,8 +145,12 @@ auto Transporter<M>::operator()(SpanConstPrimary primaries) -> TransporterResult
 #else
     ScopedSignalHandler interrupted{SIGINT};
 #endif
-    CELER_LOG_LOCAL(status)
-        << "Transporting " << primaries.size() << " primaries";
+
+    auto const evt_id = primaries.front().event_id;
+    if (evt_id.get() % print_progress_ == 0)
+    {
+        progress(evt_id, primaries.size());
+    }
```

Comment:
Since we're planning to get rid of the "event ID" part of the primaries at some point (a group of primaries should always be from the same event, and it's just going to be input to the track initializers) we might even want to just have an internal counter.
But anyway, since `evt_id` is just used inside the conditional you can use this C++17 syntax (and add a check for the event ID being set I suppose)
```suggestion
    if (auto const evt_id = primaries.front().event_id;
        evt_id && evt_id.get() % print_progress_ == 0)
    {
        progress(evt_id, primaries.size());
    }
```